packages/api/test/e2e/nft.e2e.ts

```diff
@@ -64,6 +66,41 @@ afterAll(async () => {
 describe('NFTs', () => {
   let collectionId: number, collectionId2: number;
 
+  beforeEach(async done => {
```

Comment:
do we want create a collection and mint series - beforeAll (one time) or beforeEach...
BeforeEach will end up creating collection before each test?

Reply:
Yeah I guess my thinking is that each test should really have their own collection and series etc incase they modify it during the test then it won't effect other tests. However right now we have a lot of interdependency between tests which isn't great. 

Thought it might be worth leaving in `BeforeEach` even though only one test really utilises it so we have example of how to write subsequent tests and possibly refactor the current in the future.